### PR TITLE
[DI2-368] Fixed legend login link color

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: src/map/components/legend/MapLegend.jsx [hunk 1]
```diff
@@ -31,6 +31,18 @@ const NonSelectableLegendParagraph = styled.p`
   margin-bottom: ${themeSpacing(2)};
 `
 
+const Notification = styled.div`
```
Comment (jonkoops) [inline src/map/components/legend/MapLegend.jsx:34]: Is this something we could possibly replace with the [Alert](https://amsterdam.github.io/amsterdam-styled-components/?path=/docs/ui-alert--variants) component?

**Review by jonkoops — APPROVED** [review]

**Review by chrisvanmook — APPROVED** [review]

File: src/map/components/legend/MapLegend.jsx (hunk 1 above)
Comment (alimpens) [reply src/map/components/legend/MapLegend.jsx:34]: Yes, that makes sense, I'll change it

Comment (alimpens) [reply src/map/components/legend/MapLegend.jsx]: I replaced the styled div with the Alert component. This does mean that the notification is focusable now, without it being interactive (only the link inside the notification is interactive). Is that ok, a11y-wise?

**Review by jonkoops — APPROVED** [review]

File: src/map/components/legend/MapLegend.jsx (hunk 1 above)
Comment (jonkoops) [reply src/map/components/legend/MapLegend.jsx]: Yeah, this makes sense. It can be focused so that the screen reader can read it when navigating around.

**Pull request merged**